html/contributors.html

```diff
@@ -0,0 +1,346 @@
+                <div class="card-body d-flex flex-column">
+                    <img src="../img/LogoUN.png" class="img-circle" width="100" height="100">
+                    <h5 class="card-title">Daniel Castro Alvarado</h5>
+                    <h6 class="card-subtitle mb-2 text-muted">Card subtitle</h6>
+                    <p class="card-text">Some quick example text to build on the card title and make up the bulk of the
+                        card's content.</p>
+                    <div class="mt-auto mx-1">
+                        <a href="https://github.com/0xDCA" class="mr-2 text-dark lead"><i
+                                class="icon ion-logo-github "></i></a>
+                    </div>
+                </div>
+            </div>
+        </li>
+        <li class="list-inline-item">
+            <div class="card ml-2 my-2" style="width: 22.5rem; min-height: 21rem">
+                <div class="card-body d-flex flex-column">
+                    <img src="../img/LogoUN.png" class="img-circle" width="100" height="100">
+                    <h5 class="card-title">Juan David Valencia</h5>
+                    <h6 class="card-subtitle mb-2 text-muted">Card subtitle</h6>
+                    <p class="card-text">Some quick example text to build on the card title and make up the bulk of the
+                        card's content.</p>
+                    <div class="mt-auto mx-1">
+                        <a href="https://github.com/amicis31" class="mr-2 text-dark lead"><i
+                                class="icon ion-logo-github "></i></a>
+                    </div>
+                </div>
+            </div>
+        </li>
+        <li class="list-inline-item">
+            <div class="card ml-2 my-2" style="width: 22.5rem; min-height: 21rem">
+                <div class="card-body d-flex flex-column">
+                    <img src="../img/LogoUN.png" class="img-circle" width="100" height="100">
+                    <h5 class="card-title">Andres Mauricio Rondon Patiño</h5>
+                    <h6 class="card-subtitle mb-2 text-muted">Card subtitle</h6>
+                    <p class="card-text">Some quick example text to build on the card title and make up the bulk of the
+                        card's content.</p>
+                    <div class="mt-auto mx-1">
+                        <a href="https://github.com/amrondonp" class="mr-2 text-dark lead"><i
+                                class="icon ion-logo-github "></i></a>
+                    </div>
+                </div>
+            </div>
+        </li>
+        <li class="list-inline-item">
+            <div class="card ml-2 my-2" style="width: 22.5rem; min-height: 21rem">
+                <div class="card-body d-flex flex-column">
+                    <img src="../img/LogoUN.png" class="img-circle" width="100" height="100">
+                    <h5 class="card-title">andrescorso</h5>
+                    <h6 class="card-subtitle mb-2 text-muted">Card subtitle</h6>
```

Comment:
```suggestion
                    <h6 class="card-subtitle mb-2 text-muted">Estudiante de Maestría en Ingeniería de Sistemas y Computación</h6>
```
I not quite sure, @ferestrepoca can confirm this.

Reply:
Yes, he is.